src/api/mockApi/mockApi.js

```diff
@@ -37,6 +38,8 @@ noqMockApi.onPost("api/login/").reply((config) => {
     login_status: true,
     message: "Login Successful",
     groups: ["user"],
+    first_name:"",
```

Comment:
You should use:
For User -> first_name = "User", last_name="User"
For Host -> first_name= "User", last_name="Host"
For Caseworker -> first_name = "User", last_name="Caseworker"

so first_name is the same for all, last_name is different. There is some if clauses below the row 45 you need to utilize.